app/controllers/dialcode/DialcodeV3Controller.java

```diff
@@ -76,7 +76,12 @@ public Promise<Result> searchDialCode() {
         Request request = getRequest();
         try {
             Map<String, Object> map = (Map<String, Object>) request.get(DialCodeEnum.search.name());
-            Response response = dialCodeManager.searchDialCode(request.getContext(), map);
+            List<String> fieldsList = null;
+            if(request.get(DialCodeEnum.fields.name()) != null) {
+                fieldsList = (List<String>) request.get(DialCodeEnum.fields.name());
+                fieldsList.add("identifier");
+            }
```

Comment:
What if you get a string in "fields"?

Reply:
That is returning ClassCastException. Added this code as a good to have. If this is not required, will remove the same.

Reply:
What is the API returning when it throws ClassCastException?

Reply:
![image](https://user-images.githubusercontent.com/19702246/223346551-d7324f15-f7b3-4dfa-b0b7-399b2229dfa0.png)

Reply:
We should handle this scenario and return client error or accept the string format and covert to list internally. It should not return an Internal Server Error. Please handle.